app/api/case/controller.js

```diff
@@ -0,0 +1,191 @@
+// app/api/case/controller.js
+
+const casesService = require('../../../db/models/cases');
+const organizationsService = require('../../../db/models/organizations');
+const publicationsService = require('../../../db/models/publications');
+const utils = require('../../lib/utils');
+const publicationFiles = require('../../lib/publicationFiles');
+const writePublishedFiles = require('../../lib/writePublishedFiles');
+const writeToGCSBucket = require('../../lib/writeToGCSBucket');
+
+/**
+ * @method fetchCasePoints
+ *
+ * Returns all points of concern for the provided case.
+ *
+ */
+exports.fetchCasePoints = async (req, res) => {
+  const { caseId } = req.query;
+
+  if (!caseId) throw new Error('Case ID is not valid.')
+
+  let concernPoints = await casesService.fetchCasePoints(caseId)
+  if (concernPoints) {
+    res.status(200).json({ concernPoints });
+  }
+  throw new Error('Internal server error.');
+};
+
+/**
+ * @method createCasePoint
+ *
+ * Creates a new point of concern to be associated with the case.
+ *
+ */
+exports.createCasePoint = async (req, res) => {
+  const { caseId, point } = req.body;
+
+  if (!caseId) throw new Error('Case ID is not valid.')
+  if (!point.latitude) throw new Error('Latitude is not valid.')
+  if (!point.longitude) throw new Error('Latitude is not valid.')
+  if (!point.time) throw new Error('Latitude is not valid.')
+
+  let concernPoint = await casesService.createCasePoint(caseId, point)
+  if (concernPoint) {
+    res.status(200).json({ concernPoint });
+  }
+  throw new Error('Internal server error.');
+};
+
+/**
+ * @method consentToPublish
+ *
+ * Captures user consent to having their data published in the 
+ * aggregated anonymized JSON file that is available to public.
+ *
+ */
+exports.consentToPublish = async (req, res) => {
+  const { caseId } = req.body;
+
+  if (!caseId) throw new Error('Case ID is not valid.')
+
+  const data = {
```

Comment:
This endpoint doesn't seem to actually persist anything? I realize you might be planning taking care of that at a later date. If so, mind adding a `TODO`?

Reply:
Planning to handle later once the API spec is finalized for it.  Is currently in a "Needs Confirmation" state

Reply:
Will add todo.

Reply:
Done